Localizations/nl.lproj/Localizable.strings

```diff
@@ -57,8 +57,8 @@
 "Unlock port" = "Laadpoort openen";
 "%@ port" = "Laadpoort %@";
 "Homelink" = "Homelink"; //No Machine Translation
-"Disable Sentry" = "Sentrymode Uit";
-"Enable Sentry" = "Sentrymode Aan";
+"Disable Sentry" = "Sentry Uit"; // this needs to be really short to fit in buttonbar
```

Comment:
I'll have a look in the app - I would prefer localizations to not have to take rendering into account, so if there are rendering issues, I'd rather fix them generically so all languages are fixed - and not depend on sub-optimal translations.

Reply:
Thanks, that clarifies a lot. You can delete this change (or I'll fix it in the next pull)